snapshots/un/2023-10-09/plastic_waste_1999_2010.csv.dvc

```diff
@@ -0,0 +1,40 @@
+# Learn more at:
+# http://docs.owid.io/projects/etl/architecture/metadata/reference/origin/
+meta:
+  origin:
+    # Data product / Snapshot
+    title: 'UN Comtrade: Plastic Waste Trade (UN, 2023)'
+    description: |-
+      The United Nations Comtrade database aggregates detailed global annual and monthly trade statistics by product and trading partner for use by governments, academia, research institutes, and enterprises. Data compiled by the United Nations Statistics.
+
+      Division covers approximately 200 countries and represents more than 99% of the world's merchandise trade. Information can be extracted in a variety of formats, including API developer tools for integration into enterprise applications and workflows.
+
+      Subscribers receive access to additional functionality to improve efficiency and specificity.
+
+
+      The conversion factors used in Comtrade are domestic currency per US Dollar (period average) from IMF.
+
+
+          Contact subscriptions@un.org for information and licensing.
+    date_published: latest
```

Comment:
Although permitted, I'd avoid using `data_published: latest`. In this case, I'd use the year alone because the date might not be available, e.g.

```yaml
date_published: 2023
```